hazelcast/src/main/java/com/hazelcast/concurrent/lock/operations/LockOperation.java

```diff
@@ -44,7 +45,25 @@ public LockOperation(ObjectNamespace namespace, Data key, long threadId, long le
 
     @Override
     public void run() throws Exception {
-        response = getLockStore().lock(key, getCallerUuid(), threadId, getReferenceCallId(), leaseTime);
+        ILogger logger = getLogger();
+
+        if (logger.isFinestEnabled()) {
```

Comment:
I think it is enough to log the end result and skip logging the intent (`Acquiring lock...`). Same for the `UnlockOperation`.